backend/side_stacker/game/models.py

```diff
@@ -17,37 +18,47 @@ class Game(models.Model):
     board = models.JSONField(default=initial_board)
     current_player = models.CharField(max_length=1, default='X')
     winner_player = models.CharField(max_length=1, null=True)
+    players = models.IntegerField(default=2)
 
     def execute_movement(self, movement, player):
         row_num = movement['row']
         direction = movement['direction']
         row = self.board[row_num]
         if self.current_player == player:
-            empty_index = None
-            if direction == 'L':
-                i = 0
-                while i < len(row):
-                    piece = row[i]
-                    if piece == 0:
-                        empty_index = i
-                        break
-                    i += 1
-            else:
-                i = len(row) - 1
-                while i >= 0:
-                    piece = row[i]
-                    if piece == 0:
-                        empty_index = i
-                        break
-                    i -= 1
+            empty_index = self.check_if_empty_index(direction, row)
```

Comment:
Refactor to a function that discover if the with that direction and row we find an empty index